Tests/KernelTests/AccessValidator/Consumers/CheckRightsServiceConsumerTests.cs

```diff
@@ -0,0 +1,136 @@
+﻿using LT.DigitalOffice.CheckRightsService.Repositories.Interfaces;
+using LT.DigitalOffice.Kernel.AccessValidator.Requests;
+using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.Kernel.UnitTestLibrary;
+using MassTransit;
+using MassTransit.Testing;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using CRS = LT.DigitalOffice.CheckRightsService.Broker.Consumers;
+
+namespace KernelUnitTests.AccessValidator.Consumers
+{
+    public class CheckRightsServiceConsumerTests
+    {
+        private ConsumerTestHarness<CRS.AccessValidatorConsumer> consumerHarness;
+        private InMemoryTestHarness harness;
+        private Mock<ICheckRightsRepository> repositoryMock;
+        private IRequestClient<IAccessValidatorRequest> requestClient;
```

Comment:
Can be separated declaration Mock variable from usual

Reply:
Will fix

Reply:
Fixed